packages/workspace/src/command-line/shared.ts

```diff
@@ -388,7 +401,7 @@ export function readNxJson(): NxJson {
 
 export const getAffected = (affectedNamesFetcher: AffectedFetcher) => (
   touchedFiles: string[]
-): string[] => {
+): AffectedProject[] => {
```

Comment:
see the commend about the graph.

I'd like all of these return the graph with "highlighted" nodes and the roots.